app/javascript/App/components/ItemSelectorAlt.js

```diff
@@ -44,10 +45,14 @@ export default class ItemSelectorAlt extends Component {
       )
     } else {
       return (
-        <a className="mid-gray" onClick={() => this.add(item)}>
-          <input type="checkbox" className="mr2 w-auto" />
+        <label
```

Comment:
This is good, but the same `<label>` wrap needs to be applied if the item is selected.

Reply:
nice find!